.eslintrc.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
 				' * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license',
 				' */'
 			]
-		} ]
+		} ],
+		'ckeditor5-rules/use-require-for-debug-mode-imports': 'error'
```

Comment:
Shouldn't we add this rule by default in the `eslint-config-ckeditor5` package? Other default rules are already there: https://github.com/ckeditor/ckeditor5-linters-config/blob/master/packages/eslint-config-ckeditor5/.eslintrc.js#L305-L307

Reply:
Added by @pomek. Removing the rule from here.